docs/src/pages/component-demos/drawers/UndockedDrawer.js

```diff
@@ -26,28 +26,217 @@ const styleSheet = createStyleSheet('UndockedDrawer', () => ({
     width: 250,
     flex: 'initial',
   },
+  listFull: {
+    width: 'auto',
+    flex: 'initial',
+  },
   remainder: {
     flex: 1,
   },
 }));
 
 export default class UndockedDrawer extends Component {
   state = {
-    open: false,
+    open: {
+      top: false,
+      left: false,
+      bottom: false,
+      right: false,
+    },
   };
 
-  handleOpen = () => this.setState({ open: true });
-  handleClose = () => this.setState({ open: false });
+  toggleDrawer = (side, open) => {
+    const drawerState = {};
+    drawerState[side] = open;
+    this.setState({ open: drawerState });
+  };
+
+  handleTopOpen = () => this.toggleDrawer('top', true);
+  handleTopClose = () => this.toggleDrawer('top', false);
+  handleLeftOpen = () => this.toggleDrawer('left', true);
+  handleLeftClose = () => this.toggleDrawer('left', false);
+  handleBottomOpen = () => this.toggleDrawer('bottom', true);
+  handleBottomClose = () => this.toggleDrawer('bottom', false);
+  handleRightOpen = () => this.toggleDrawer('right', true);
+  handleRightClose = () => this.toggleDrawer('right', false);
 
   render() {
     const classes = this.context.styleManager.render(styleSheet);
     return (
       <div>
-        <Button onClick={this.handleOpen}>Open Drawer</Button>
+        <Button onClick={this.handleTopOpen}>Open Top Drawer</Button>
```

Comment:
I would have the left drawer first in the list as people will most likely try that one first.